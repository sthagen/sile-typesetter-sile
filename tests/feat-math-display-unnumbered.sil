\begin[papersize=a6]{document}
\nofolios
\neverindent
\use[module=packages.lorem]
\use[module=packages.math]
\font[size=11pt]
% Test default (unnumbered) display math
% within a blockquote (to check it honors margin skips)
% and in all alignments

\begin{blockquote}
Begin justified
\math[mode=display]{e^{i\pi} = -1}
End justified.

\begin{center}
Begin centered
\math[mode=display]{e^{i\pi} = -1}
End centered
\end{center}

\begin{raggedright}
Begin left-aligned
\math[mode=display]{e^{i\pi} = -1}
End left-aligned.
\end{raggedright}

\begin{raggedleft}
Begin right-aligned
\math[mode=display]{e^{i\pi} = -1}
End right-aligned
\end{raggedleft}
\end{blockquote}

\end{document}
