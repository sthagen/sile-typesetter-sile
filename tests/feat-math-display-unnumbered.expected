Set paper size 	297.6377985	419.5275636
Begin page
Mx 	36.8819
My 	28.8853
Set font 	Gentium Plus;10.45;400;;normal;;;LTR
T	37 72 74 76 81 w=24.2677	(Begin)
Mx 	63.4695
T	77 88 86 87 76 737 72 71 w=35.0493	(justified)
Mx 	131.9113
My 	51.4155
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	2750 w=3.8100	(nil)
Mx 	135.7213
My 	47.6655
Set font 	Libertinus Math;8;400;Regular;normal;+ssty=1;;LTR
T	2753 w=1.8320	(nil)
Mx 	137.5533
T	3421 w=4.5776	(nil)
Mx 	145.3387
My 	51.4155
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	30 w=6.4800	(nil)
Mx 	154.5965
T	2088 w=6.4800	(nil)
Mx 	161.0765
T	18 w=4.6500	(nil)
Mx 	36.8819
My 	73.9458
Set font 	Gentium Plus;10.45;400;;normal;;;LTR
T	40 81 71 w=16.3843	(End)
Mx 	55.5867
T	77 88 86 87 76 737 72 71 w=35.0493	(justified)
Mx 	90.6360
T	17 w=2.3931	(.)
Mx 	116.5428
My 	86.4858
T	37 72 74 76 81 w=24.2677	(Begin)
Mx 	143.1117
T	70 72 81 87 72 85 72 71 w=37.9833	(centered)
Mx 	131.9113
My 	109.0160
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	2750 w=3.8100	(nil)
Mx 	135.7213
My 	105.2660
Set font 	Libertinus Math;8;400;Regular;normal;+ssty=1;;LTR
T	2753 w=1.8320	(nil)
Mx 	137.5533
T	3421 w=4.5776	(nil)
Mx 	145.3387
My 	109.0160
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	30 w=6.4800	(nil)
Mx 	154.5965
T	2088 w=6.4800	(nil)
Mx 	161.0765
T	18 w=4.6500	(nil)
Mx 	120.4845
My 	131.5463
Set font 	Gentium Plus;10.45;400;;normal;;;LTR
T	40 81 71 w=16.3843	(End)
Mx 	139.1700
T	70 72 81 87 72 85 72 71 w=37.9833	(centered)
Mx 	36.8819
My 	144.0863
T	37 72 74 76 81 w=24.2677	(Begin)
Mx 	63.4508
T	79 72 73 87 w=14.5065	(left)
Mx 	77.9573
T	16 w=3.5208	(-)
Mx 	81.4781
T	68 79 76 74 81 72 71 w=31.5847	(aligned)
Mx 	131.9113
My 	166.6165
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	2750 w=3.8100	(nil)
Mx 	135.7213
My 	162.8665
Set font 	Libertinus Math;8;400;Regular;normal;+ssty=1;;LTR
T	2753 w=1.8320	(nil)
Mx 	137.5533
T	3421 w=4.5776	(nil)
Mx 	145.3387
My 	166.6165
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	30 w=6.4800	(nil)
Mx 	154.5965
T	2088 w=6.4800	(nil)
Mx 	161.0765
T	18 w=4.6500	(nil)
Mx 	36.8819
My 	189.1467
Set font 	Gentium Plus;10.45;400;;normal;;;LTR
T	40 81 71 w=16.3843	(End)
Mx 	55.5674
T	79 72 73 87 w=14.5065	(left)
Mx 	70.0739
T	16 w=3.5208	(-)
Mx 	73.5947
T	68 79 76 74 81 72 71 w=31.5847	(aligned)
Mx 	105.1794
T	17 w=2.3931	(.)
Mx 	177.6509
My 	201.6867
T	37 72 74 76 81 w=24.2677	(Begin)
Mx 	204.2198
T	85 76 74 75 87 w=21.4307	(right)
Mx 	225.6504
T	16 w=3.5208	(-)
Mx 	229.1712
T	68 79 76 74 81 72 71 w=31.5847	(aligned)
Mx 	131.9113
My 	224.2170
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	2750 w=3.8100	(nil)
Mx 	135.7213
My 	220.4670
Set font 	Libertinus Math;8;400;Regular;normal;+ssty=1;;LTR
T	2753 w=1.8320	(nil)
Mx 	137.5533
T	3421 w=4.5776	(nil)
Mx 	145.3387
My 	224.2170
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	30 w=6.4800	(nil)
Mx 	154.5965
T	2088 w=6.4800	(nil)
Mx 	161.0765
T	18 w=4.6500	(nil)
Mx 	185.5343
My 	246.7472
Set font 	Gentium Plus;10.45;400;;normal;;;LTR
T	40 81 71 w=16.3843	(End)
Mx 	204.2198
T	85 76 74 75 87 w=21.4307	(right)
Mx 	225.6504
T	16 w=3.5208	(-)
Mx 	229.1712
T	68 79 76 74 81 72 71 w=31.5847	(aligned)
End page
Finish
